soc/spid.c

```diff
@@ -108,8 +108,7 @@ struct pollfd fds[N_POLLFDS];
 int usbd_sock_fd;
 struct sockaddr_un usbd_sock_addr;
 void delay() {
-    volatile int i = 1000;
-    while(i--);
```

Comment:
Was that before I fixed the kernel bug making timing unpredictable? In any case, I think we need a fairly long delay here when all three channels are doing things in the completion handler, but don't have any reasoning for how long besides "I stop seeing duplicated and missing packets with this change", which isn't a great way to do it. But I'd rather have it slow and working than fast and broken and it can be optimized later.